spec/unit/recipes/_configure_spec.rb

```diff
@@ -1,14 +1,14 @@
-# coding: UTF-8
+# frozen_string_literal: true
 
```

Comment:
There's a warning on line 83 of this file.  You can pull over this line from #155 to resolve it: https://github.com/cerner/cerner_splunk/pull/155/files#diff-2c8e2001d0880b470cbfaef7c90399c4R83

Reply:
I don't have any warning

Reply:
```
$ rspec spec/unit/recipes/_configure_spec.rb 

Randomized with seed 5137

cerner_splunk::_configure
  when cluster databag is specified
    includes cerner_splunk::_configure_authentication recipe
    includes cerner_splunk::_configure_inputs recipe
    includes cerner_splunk::_configure_apps recipe
    includes cerner_splunk::_configure_outputs recipe
    includes cerner_splunk::_configure_alerts recipe
    includes cerner_splunk::_configure_server recipe
    includes cerner_splunk::_configure_roles recipe
  when cluster is not specified
    when node is a forwarder
      should raise a warning message
    when node is not a forwarder
WARNING: Using the `raise_error` matcher without providing a specific error or message risks false positives, since `raise_error` will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`, potentially allowing the expectation to pass without even executing the method you are intending to call. Actual error raised was #<RuntimeError: You need to configure at least one cluster databag.>. Instead consider providing a specific error class or message. This message can be suppressed by setting: `RSpec::Expectations.configuration.on_potential_false_positives = :nothing`. Called from /Users/bg010740/.rvm/gems/ruby-2.3.1/gems/chefspec-7.1.0/lib/chefspec/expect_exception.rb:15:in `matches?'.
      should raise an exception
```

Reply:
aa57872a25700a2dcd3b344f96fcec5afc72f554